code/sha3/Hacl.Hash.SHA3.Simd256.fst

```diff
@@ -27,6 +27,17 @@ module M = LowStar.Modifies
 
 #reset-options "--z3rlimit 50 --max_fuel 0 --max_ifuel 0 --using_facts_from '* -FStar.Seq'"
 
+val absorb_inner_256:
+    rateInBytes:size_t{v rateInBytes > 0 /\ v rateInBytes <= 200}
+  -> b: Hacl.Impl.SHA2.Types.bufx4
```

Comment:
this it the important bit that encodes that SHA3.Simd256 now depends on the types (and pretty names) contains in that file -- this instructs krml to use those pretty names when monomorphizing types.